packages/cadl-ranch-specs/http/type/property/nullable/main.tsp

```diff
@@ -147,3 +147,17 @@ model CollectionsModelProperty is ModelTemplate<InnerModel[]>;
 @operationGroup
 interface CollectionsModel
   extends OperationsTemplate<CollectionsModelProperty, "[{'property': 'hello'}, {'property': 'world'}]"> {}
+
+// Model with nullable collection int32 property
+@doc("Model with collection int32 properties")
+model CollectionsInt32Property is ModelTemplate<int32[]>;
+@route("/collections/int32")
+@operationGroup
+interface CollectionsInt32 extends OperationsTemplate<CollectionsInt32Property, "[123, 456]"> {}
+
+// Model with nullable collection string property
+@doc("Model with collection string properties")
+model CollectionsStringProperty is ModelTemplate<string[]>;
+@route("/collections/string")
+@operationGroup
+interface CollectionsString extends OperationsTemplate<CollectionsStringProperty, "['hello', 'world']"> {}
```

Comment:
The test here is whether user need to send an array or no array (null or absent). Why make 2 test case about content of the array? It should not matter at all.

Reply:
fixed

Reply:
What I mean is that a test with `string[] | null` should already do. No need for anther `int32[] | null` (if you have `int32` then why not also `float32`? I think it is useless repetitions)

Reply:
fixed